packages/shared/src/components/modals/BootPopups.tsx

```diff
@@ -248,28 +248,9 @@ export const BootPopups = (): ReactElement => {
     addBootPopup({
       type: LazyModal.TopReaderBadge,
       props: {
-        onAfterClose: (keywordValue: string) => {
+        onAfterClose: () => {
```

Comment:
Oh it's now like a extra afterClose?

Reply:
Most of it's now inside the `TopReaderBadgeModal` component, so we don't need to duplicate the logEvent calls. Then we can just set onAfterClose/ onAfterOpen props if we need anything extra.
I.e. in BootPopups we want to clear alerts after closing, and on notifications page want to revert the URL.

Reply:
Just makes it more dynamic, but keeps the common things in one place.